daml-lf/encoder/BUILD.bazel

```diff
@@ -10,6 +10,13 @@ load(
     "lf_scalacopts",
 )
 
+lf_target_versions = [
```

Comment:
I suspect you can use the keys of lf_since_deps instead, perhaps moving it up here.

Reply:
fixed